pkg/log/fiano_logger.go

```diff
@@ -0,0 +1,32 @@
+package log
+
+import (
+	"github.com/facebookincubator/go-belt/tool/logger"
+	fianoLog "github.com/linuxboot/fiano/pkg/log"
+)
+
+var _ fianoLog.Logger = FianoLogger{}
+
+// FianoLogger is just a placeholder for a logger, which does nothing
+type FianoLogger struct {
+	Backend    logger.Logger
+	LogAsLevel logger.Level
```

Comment:
Do these need to be exported? I don't see them used outside this package?

Reply:
Any reason to hide it? :)

Reply:
I generally lean towards not exporting things that aren't needed so interface boundaries are kept to only the required pieces, but no strong push towards that here given it's all internal.

Reply:
It seems to me C++-y approach. I'd prefer in Go to export by default if there are no explicit safety concerns. In my opinion in Go we need to protect fields only if there are hidden assumptions about these fields.

Reply:
Not insisting here, but let's have a conversation about it when I'm back after PTO.